### DISCUSSION TITLE
My student devloper pack is approved but still I didn't get my benefit 😕😕

### DISCUSSION THREAD

**Discussion by HackerAditya56** [community/community#138077] [GitHub Education] [answered]
### Hi everyone,

I applied for students devloper pack and it's got approved but still after 4 days I didn't receive any email or neither the benefits 😕😕😕

**Comment by Sai1234567890123** [comment]
I applied for students devloper pack and it's got approved but still after 3days I didn't receive any email or neither the benefits 😕😕😕

**Reply by ElephantFire** [reply]
me too

**Comment by synasapmob** [comment]
check in your **'spam email'** again

**Comment by davevad93** [comment, marked as answer]
Hi @HackerAditya56 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @HackerAditya56 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @HackerAditya56 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

**Reply by HackerAditya56** [reply]
I am getting a issue with Microsoft azure help me

**Reply by davevad93** [reply]
Hey @HackerAditya56 , check your other open discussion #138095 , I answered you there, but note that if your benefits aren't yet available you can't redeem your Azure offer.

**Comment by Manpreet0980** [comment]
I am approved my GitHub application but benefits not give so plz solve
